virtual_rainforest/models/soil/carbon.py

```diff
@@ -51,46 +56,61 @@ def calculate_soil_carbon_updates(
         A vector containing net changes to each pool. Order [lmwc, maom].
     """
 
-    # Find scalar factors that multiple rates
-    temp_scalar = convert_temperature_to_scalar(
-        soil_temp,
-        constants.temp_scalar_coefficient_1,
-        constants.temp_scalar_coefficient_2,
-        constants.temp_scalar_coefficient_3,
-        constants.temp_scalar_coefficient_4,
-        constants.temp_scalar_reference_temp,
-    )
+    # TODO - At present we have two different factors that capture the impact of soil
+    # water. water_factor is based on soil water potential and applies to the microbial
+    # processes, whereas moist_scalar is based on soil water content and applies to the
+    # chemical processes. In future, all processes will be microbially mediated, meaning
+    # that moist_scalar can be removed.
+
+    # Find the impact of soil water content on chemical soil processes
     moist_scalar = convert_moisture_to_scalar(
         soil_moisture,
         constants.moisture_scalar_coefficient,
         constants.moisture_scalar_exponent,
     )
-    moist_temp_scalar = moist_scalar * temp_scalar
-
+    # Find the impact of soil water potential on the biochemical soil processes
+    water_factor = calculate_water_potential_impact_on_microbes(
+        water_potential=soil_water_potential,
+        water_potential_halt=constants.soil_microbe_water_potential_halt,
+        water_potential_opt=constants.soil_microbe_water_potential_optimum,
+        moisture_response_curvature=constants.moisture_response_curvature,
+    )
     # Calculate transfers between pools
     lmwc_to_maom = calculate_mineral_association(
         soil_c_pool_lmwc,
         soil_c_pool_maom,
         pH,
         bulk_density,
-        moist_temp_scalar,
+        moist_scalar,
         percent_clay,
         constants,
     )
```

Comment:
For consistency, could you add the argument names here, too? For example: `moist_scalar=moist_scalar`